src/test/java/springsamurais/toyswapbackend/repository/ListingRepositoryTest.java

```diff
@@ -2,6 +2,6 @@
 
 import static org.junit.jupiter.api.Assertions.*;
 
-class PostInterfaceTest {
+class ListingRepositoryTest {
 
```

Comment:
I don't think repository tests are necessary, might be good with time but these aren't critical.